CHANGES.rst

```diff
@@ -141,6 +141,8 @@ astropy.stats
 
 astropy.table
 ^^^^^^^^^^^^^
+- ``Table.from_pandas`` no supports a ``units`` dictionary as argument to pass units
```

Comment:
Only one tiny problem here, `no` should be `now`.  You can fix this and include `[docs only]` in the commit message to avoid doing all the CI code tests.